providers/org/arxiv/harvester.py

```diff
@@ -1,7 +1,66 @@
-from share.harvest.harvester import OAIHarvester
+import logging
+import datetime
 
+from furl import furl
+from lxml import etree
 
-class ArxivHarvester(OAIHarvester):
-    rate_limit = (1, 3)
-    time_granularity = False
-    url = 'http://export.arxiv.org/oai2'
+from share import Harvester
+
+logger = logging.getLogger(__name__)
+
+
+class ArxivHarvester(Harvester):
+    namespaces = {
+        'ns0': 'http://www.w3.org/2005/Atom'
+    }
+    start_page_num = 0
+    url = 'http://export.arxiv.org/api/query?search_query=all&max_results=100&sortBy=lastUpdatedDate&sortOrder=descending'
+
+    def get_next_url(self):
+        self.start_page_num += 100
+        return self.url + '&start={}'.format(self.start_page_num)
```

Comment:
Use furl here just for consistency 
@caseyrollins has a pretty good example https://github.com/CenterForOpenScience/SHARE/pull/187/files#diff-7d27ce3f32de285c1155f8d2b7fc9734R22